public/content/documentation/web/component/informative-image.md

```diff
@@ -6,75 +6,140 @@ How to test an informative image
 
 2. Test mobile screenreader gestures
 
-   - Swipe: Focus moves directly to the header or navigation
-
-   - Doubletap: This typically activates most elements
+   - Swipe: The screenreader reads the alt text (ignored if decorative)
 
 3. Listen to screenreader output on all devices
 
-   - It is discoverable with screenreader shortcuts as header/banner landmark
-
-   - Group: It typically contains the name and primary navigation of the website
+   - Name: The content of the image alt text is clear (ignored if decorative)
+   - Role: It identifies its role as an image or graphic (ignored if decorative)    
 
 Full information: [https://www.magentaa11y.com/MagentaA11yV2#/web-criteria/component/informative-image](https://www.magentaa11y.com/MagentaA11yV2#/web-criteria/component/informative-image)
 
 ## Gherkin
 
 ### #a11y - Web Accessibility Acceptance Criteria
 
-How to test a header
+How to test an informative image
 
-GIVEN THAT I am on a page with a header landmark
+GIVEN THAT I am on a page with an informative image
 
 1. Keyboard for mobile & desktop
 
-   - WHEN I use the tab key to enter the web browser window I SEE focus is strongly visually indicated on interactive components
+   - WHEN I use the arrow keys to browse to an image I SEE the image comes into view
 
 2. Desktop screenreader
 
-   - WHEN I use a desktop screenreader (NVDA, JAWS, VoiceOver) AND
+   - WHEN I use a desktop screenreader (NVDA, JAWS, VoiceOver) AND I use the arrow keys to browse to an image
+      - I HEAR the content of the image alt text is clear (ignored if decorative)
+      - I HEAR it identifies its role as an image or graphic (ignored if decorative)
 
-   - I use the tab key to enter the web browser window
+3. Mobile screenreader
 
-   - I HEAR It is discoverable with screenreader shortcuts as header/banner landmark
+   - WHEN I use a mobile screenreader (Talkback, VoiceOver) AND I swipe to browse to an image
+      - I HEAR the content of the image alt text is clear (ignored if decorative)
+      - I HEAR it identifies its role as an image or graphic (ignored if decorative)
 
-   - I HEAR It typically contains the name and primary navigation of the website
 
-3. Mobile screenreader
+Full information: [https://www.magentaa11y.com/MagentaA11yV2#/web-criteria/component/informative-image](https://www.magentaa11y.com/MagentaA11yV2#/web-criteria/component/informative-image)
 
-   - WHEN I use a mobile screenreader (Talkback, VoiceOver) AND
+## Is this image decorative or informative?
+If the image conveys important meaning, and there's no other text on the page which explains the concept within it, then the image is likely informative. If the image is included for purely stylistic purposes and doesn't impart any meaning to the rest of the content on the page, then the image is likely decorative. In this case, check out the [decorative image checklist](/checklist-web/image-decorative) item instead. 
```

Comment:
we'll have to update this link for decorative image to the entry on the new site